src/mlpack/core/data/split_data.hpp

```diff
@@ -98,21 +104,27 @@ void Split(const arma::Mat<T>& input,
  * @param trainData Matrix to store training data into.
  * @param testData Matrix to store test data into.
  * @param testRatio Percentage of dataset to use for test set (between 0 and 1).
+ * @param shuffleData True(Default) if you want to shuffle data.
```

Comment:
What about: "If true, the sample order is shuffled; otherwise, each sample is visited in linear order. (Default true)."

Reply:
Done